virttest/utils_misc.py

```diff
@@ -2107,16 +2107,17 @@ def get_cpu_vendor(cpu_info="", verbose=True):
 
 def get_support_machine_type(qemu_binary="/usr/libexec/qemu-kvm"):
     """
-    Get the machine type the host support,return a list of machine type
+    Get the machine type the host support,return a tuple (s, c),
+    s is machine type list, and c is system version list
     """
     o = process.system_output("%s -M ?" % qemu_binary).splitlines()
     s = []
     c = []
     for item in o[1:]:
-        machine_list = re.split(r"\s+", item, 1)
-        if machine_list[0] != "none":
+        if not ("alias" in item or "none" in item):
```

Comment:
Hi @yanan-fu , there is a [case](https://polarion.engineering.redhat.com/polarion/#/project/RedHatEnterpriseLinux7/workitem?id=RHEL7-10704) needs to be booted from '-M pc', could you confirm the code with this case?

Reply:
smbios_table.machine_type  is the corresponding automation case for the polarion case you added.
traverse all the supported machine types.  
In fact, libvrt doesn't use "-M pc", it use "pc-i440fx-rhel7.5.0" for rhel7.5. 
@xiangchunfu  what do you think about remove the alias "pc" ?

Reply:
@yanan-fu 
if skip alias lines, pc also skiped too.  so above code match your request.
And,  there also unnecessary to keep list 'c' in this function.

Reply:
@xutian list c is used to match with the output of "dmidecode" in guest. it can't be removed.
You can refer the case "smbios_table", thanks a lot!

Reply:
@yanan-fu  I think alias can be removed. reason: 1. pc is the same with pc-i440fx-rhel7.5.0. To my understanding, It is usually used as test.  2. Upstream tool don't use pc directly,but It use pc-i440fx-rhel7.5.0.  so if remove alias. it don't affect end-user.